recipes/eeharvest/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "eeharvest" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eeharvest-{{ version }}.tar.gz
+  sha256: f1506ea148f15481c16a01d78dd90977a5967b38f216f2ca3005bc075f288995
+
+build:
+  skip: true  # [py<39]
```

This one can probably be noarch.

```suggestion
  noarch: python
```